HotelBooking.Data/Extensions/MailSender.cs

```diff
@@ -8,14 +7,14 @@ namespace HotelBooking.Data.Extensions;
 
 public class MailSender : IMailSender
 {
-    public async Task<bool> SendInforOfBooking(string toEmail, InforBookingResponse model)
+    public async Task<bool> SendInforOfBooking(string toEmail, string idBooking)
     {
-        
+
         var email = new MimeMessage();
         email.From.Add(MailboxAddress.Parse("girlmilk123@gmail.com"));
         email.To.Add(MailboxAddress.Parse(toEmail));
         email.Subject = "Confirm EMail";
-        email.Body = new TextPart(TextFormat.Plain) { Text = model.InformationSender()};
+        email.Body = new TextPart(TextFormat.Plain) { Text = $"https://localhost:7137/api/Booking/get-booking-by-id?id={idBooking}" };
```

Comment:
I'm not a fan of this kind. You can create a method name generateEmailBody() and update anything related to email body in this method.

Reply:
I fixed it. thanks